Solutions/Zscaler Private Access (ZPA)/Workbooks/ZscalerZPA.json

```diff
@@ -0,0 +1,323 @@
+        "size": 1,
+        "title": "Top VPN Traffic (per user)",
+        "timeContext": {
+          "durationMs": 2592000000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "tiles",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "Trend",
+              "formatter": 10,
+              "formatOptions": {
+                "palette": "blue"
+              }
+            }
+          ]
+        },
+        "tileSettings": {
+          "titleContent": {
+            "columnMatch": "User",
+            "formatter": 1,
+            "formatOptions": {}
+          },
+          "leftContent": {
+            "columnMatch": "Traffic(MB)",
+            "formatter": 12,
+            "formatOptions": {
+              "palette": "auto"
+            },
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "style": "decimal",
+                "useGrouping": false,
+                "maximumFractionDigits": 2,
+                "maximumSignificantDigits": 3
+              }
+            }
+          },
+          "secondaryContent": {
+            "columnMatch": "Trend",
+            "formatter": 10,
+            "formatOptions": {
+              "palette": "blue"
+            }
+          },
+          "showBorder": false
+        },
+        "chartSettings": {
+          "group": "User",
+          "createOtherGroup": 0
+        }
+      },
+      "name": "query - 5"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let listIPs =\nZPAEvent \n| where TimeGenerated > ago(30d)\n| where DvcAction == 'open'\n| summarize ListofIPs = make_set(SrcIpAddr) by DstUserName\n| project ListofIPs;\nlet dt_lookBack = 24h;\nZPAEvent\n| where TimeGenerated > ago(dt_lookBack)\n| where isnotempty(SrcIpAddr)\n| where SrcIpAddr !in (listIPs)\n| summarize EventCount = count() by DstUserName, SrcIpAddr\n| extend Message = \"VPN connection from new IP\"\n| project-away EventCount\n",
+        "size": 1,
+        "title": "New VPN Users",
+        "timeContext": {
+          "durationMs": 2592000000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "table"
+      },
+      "name": "query - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "SecurityAlert\r\n| where AlertName contains 'VPN'\r\n| extend Severity = AlertSeverity\r\n| extend RuleName = AlertName\r\n| project RuleName, Severity\r\n\r\n",
```

Comment:
Use HAS?

Reply:
@aprakash13 - Please review and suggest where this is pending..is it for author ?

Reply:
fixed